packages/bot-web-ui/src/components/trade-animation/trade-animation.jsx

```diff
@@ -188,4 +190,5 @@ export default connect(({ summary_card, run_panel, toolbar, ui }) => ({
     profit: summary_card.profit,
     should_show_overlay: run_panel.should_show_overlay,
     toggleAnimationInfoModal: toolbar.toggleAnimationInfoModal,
+    account_status: client.account_status.cashier_validation,
```

Comment:
For more redability, I think it's better to change the variable name to `cashier_validation ` or something like this because `account_status ` is the name of the root object

Reply:
@sara-fs oh yes, you are right, good remark👌